### PR TITLE
Update the CoreDNS version and manifest

### PR DESCRIPTION
**Pull request by rajansandeep**
**What this PR does / why we need it**:
Bumps CoreDNS to 1.1.3 and updates manifest

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
CoreDNS is now v1.1.3
```

### CODE REVIEW

**Review by MrHohn — APPROVED** [review]
/lgtm

**Comment by rajansandeep** [comment]
/assign @fabriziopandini
/retest
@luxas Want to make sure this gets in before code slush! Thanks!

File: cmd/kubeadm/app/phases/addons/dns/versions.go [hunk 1]
```diff
@@ -23,13 +23,13 @@ import (
...
-	coreDNSVersion = "1.0.6"
+	coreDNSVersion = "1.1.3"
 )
 
 // GetDNSVersion returns the right kube-dns version for a specific k8s version
 func GetDNSVersion(kubeVersion *version.Version, dns string) string {
-	// v1.9.0+ uses kube-dns 1.14.10
-	// v1.9.0+ uses CoreDNS  1.0.6 if feature gate "CoreDNS" is enabled.
+	// v1.9.0+ uses kube-dns 1.14.10, if feature gate "CoreDNS" is disabled.
+	// v1.9.0+ uses CoreDNS  1.1.3.
```
Comment (luxas) [inline cmd/kubeadm/app/phases/addons/dns/versions.go]: All kubernetes versions should use 1.1.3 you mean here, right?

**Review by luxas — APPROVED** [review]
/lgtm
/approve

Just one small question, but I guess you want all v1.10.x and v1.11.x versions to use 1.1.3

**Pull request merged**